src/main/java/Bundle.properties

```diff
@@ -2289,3 +2289,932 @@ rtabfileparser.ioexception.boolean=Unexpected value for the Boolean variable ({0
+#search-include-fragment.xhtml bundle[facetCategory.friendlyName]
+Dataverse\u0020Category=Dataverse Category
+Publication\u0020Date=Publication Date
+Author\u0020Name=Author Name
+Subject=Subject
+Deposit\u0020Date=Deposit Date
+File\u0020Type=File Type
+File\u0020Tag=File Tag
+Access=Access
+Keyword\u0020Term=Keyword Term
+Author\u0020Affiliation=Author Affiliation
+Language=Language
+Kind\u0020of\u0020Data=Kind of Data
+Publication\u0020Status=Publication Status
+Metadata\u0020Source=Metadata Source
+Harvested=Harvested
+Publication\u0020Year=Publication Year
+Topic\u0020Classification\u0020Term=Topic Classification Term
+Producer\u0020Name=Producer Name
+Production\u0020Date=Production Date
+Contributor\u0020Type=Contributor Type
+Contributor\u0020Name=Contributor Name
+Distributor\u0020Name=Distributor Name
+Distribution\u0020Date=Distribution Date
+Grant\u0020Information\u0020Grant\u0020Agency=Grant Information Grant Agency
+Grant\u0020Information\u0020Grant\u0020Number=Grant Information Grant Number
+Series\u0020Name=Series Name
+Time\u0020Period\u0020Covered\u0020Start=Time Period Covered Start
+Time\u0020Period\u0020Covered\u0020End=Time Period Covered End
+Geographic\u0020Coverage\u0020Country\u0020/\u0020Nation=Geographic Coverage Country / Nation
+Geographic\u0020Coverage\u0020State\u0020/\u0020Province=Geographic Coverage State / Province
+Geographic\u0020Coverage\u0020City=Geographic Coverage City
+Geographic\u0020Unit=Geographic Unit
+Unit\u0020of\u0020Analysis=Unit of Analysis
+Universe=Universe
+Time\u0020Method=Time Method
+Frequency=Frequency
+Response\u0020Rate=Response Rate
+Type=Type
+Facility=Facility
+Instrument=Instrument
+Object=Object
+Spatial\u0020Resolution=Spatial Resolution
+Spectral\u0020Resolution=Spectral Resolution
+Bandpass=Bandpass
+Central\u0020Wavelength\u0020(m)=Central Wavelength (m)
+Wavelength\u0020Range\u0020Minimum\u0020(m)=Wavelength Range Minimum (m)
+Wavelength\u0020Range\u0020Maximum\u0020(m)=Wavelength Range Maximum (m)
+Dataset\u0020Date\u0020Range\u0020Start=Dataset Date Range Start
+Dataset\u0020Date\u0020Range\u0020End=Dataset Date Range End
+Design\u0020Type=Design Type
+Factor\u0020Type=Factor Type
+Organism=Organism
+Other\u0020Organism=Other Organism
+Measurement\u0020Type=Measurement Type
+Other\u0020Measurement\u0020Type=Other Measurement Type
+Technology\u0020Type=Technology Type
+Technology\u0020Platform=Technology Platform
+Cell\u0020Type=Cell Type
+Journal\u0020Volume=Journal Volume
+Journal\u0020Issue=Journal Issue
+Journal\u0020Publication\u0020Date=Journal Publication Date
+Type\u0020of\u0020Article=Type of Article
+
+#Dataverse Category Facets
+Researcher=Researcher
+Research\u0020Project=Research Project
+Journal=Journal
+Organization\u0020or\u0020Institution=Organization or Institution
+Teaching\u0020Course=Teaching Course
+Research\u0020Group=Research Group
+Laboratory=Laboratory
+Department=Department
+Uncategorized=Uncategorized
+
+#astrophysics
+Image=Image
+Mosaic=Mosaic
```

Comment:
Do we really want strings that were formerly in astrophysics.tsv (for example) in the main Bundle.properties file? Should they go in src/main/java/astrophysics.properties instead? Also, does metadatacustomization.rst need to be updated to mention facets and where to put the strings when creating a custom metadata block?

Reply:
@pdurbin - The key added to the bundle.properties are the values returned from solr for facets.  Is there any way to identify which metadata property file it corresponds to?